packages/envision/src/scss/tab.scss

```diff
@@ -17,34 +17,51 @@
       }
    }
 
-   &__link {
+   &__link,
+   [role='tab'] {
       display: block;
+      align-items: center;
+      box-sizing: border-box;
+      appearance: none;
+      font-family: css-var('font-family');
+      font-size: 1em;
```

Comment:
Vill vi använda t.ex. --env-font-size-medium istället så att tabbarna skalar med temat? Nytt beteende isåfall

Reply:
Ja, vi gör ju faktiskt det på nästan alla andra komponenter. Jag ändrar.